SPECS/kernel/config

```diff
@@ -4951,6 +4952,7 @@ CONFIG_FB_SYS_IMAGEBLIT=y
 # CONFIG_FB_FOREIGN_ENDIAN is not set
 CONFIG_FB_SYS_FOPS=y
 CONFIG_FB_DEFERRED_IO=y
+CONFIG_FB_IOMEM_FOPS=y
```

Comment:
result of commit: https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/c00e8fd749502c02085534c60b1edca4fc479c91